admin/app/view_models/workarea/admin/reports/timeline_view_model.rb

```diff
@@ -32,6 +32,11 @@ def initialize(model, type:)
             @model = model
             @type = type
           end
+
+          def occurred_at
+            return @model.published_at if @type == 'release'
```

Comment:
One way around this awkwardness would be to pass in the `occured_at` when creating `Event.new`

Reply:
I wouldn't mind just aliasing `published_at` to `occured_at`. 🦆